### PR TITLE
Feature/1101 contact overflow

### PR DESCRIPTION
**Pull request by nicklee**
Refactoring opposite notch to not need overflow hidden

### CODE REVIEW

File: rca/project_styleguide/templates/patterns/atoms/section/section-example.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        </div>
+    </div>
+    <div class="section__notch">
+        <div class="section__notch-fill section__notch-fill--third-col"></div>
+    </div>
+</div>
+
+<h2 class="heading heading--two">Fourth col notch</h2>
+
+<div class="section section--start bg bg--dark">
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/atoms/section/section-example.html:74]: should this be `bg--light`?

File: rca/static_src/sass/components/_section.scss [hunk 2]
```diff
@@ -387,6 +437,26 @@
...
+                #{$root}__notch--opposite & {
+                    right: calc((var(--margin-width) * -1)); // Make negative
+                    width: calc(var(--margin-width));
+
+                    @include media-query(large) {
+                        right: calc(
+                            (
+                                    var(--margin-width) + var(--grid-width) +
+                                        var(--outer-grid-width)
+                                ) * -1
```
Comment (chris-lawton) [inline rca/static_src/sass/components/_section.scss:451]: nipick: formatting looks interesting here - guessing it's the line length rule?

**Comment by chris-lawton** [comment]
Nice work, liking the example pages!

**Review by chris-lawton — APPROVED** [review]

File: rca/static_src/sass/components/_section.scss (hunk 2 above)
Comment (nicklee) [reply rca/static_src/sass/components/_section.scss:451]: hah yea, awful isn't it, would you put an ignore in here?

Comment (chris-lawton) [reply rca/static_src/sass/components/_section.scss:451]: i think if it saved-on-format that way then i'd just leave it as is

File: rca/project_styleguide/templates/patterns/atoms/section/section-example.html (hunk 1 above)
Comment (nicklee) [reply rca/project_styleguide/templates/patterns/atoms/section/section-example.html:74]: good spot, thanks

**Pull request merged**